README.md

```diff
@@ -0,0 +1,130 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio: Capstone <a name="about-project"></a>
+
+> My poftfolio website is created using pure Html5 ,Css3 and javascript. In this project we have created a website.
+
+**Capstone first** is a Portfolio web page that can depict my coding skills
+
+<!-- BUILT WITH -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+- **HTML5** 
+- **CSS3**
+- **JavaScript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive**
+- **Dynamic**
+- **Simple**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link](https://kamranahmad123.github.io/first-capstone/)
```

Comment:
- [ ] Currently, this Live demo link is not working properly. Please add a new one that works properly 😃 👍